docs/pages/example_usage.rst

```diff
@@ -347,6 +347,12 @@ advanced search
         epsg=32648,
     )
 
+    # use ownership filters
+
+    ownedGEOItems = client.search(
```

Comment:
let's try to use snake case also for docs ;) - owned_geo_items

Reply:
Totally right, muscle memory mistake.